akka-docs/src/main/paradox/typed/fault-tolerance.md

```diff
@@ -30,10 +30,11 @@ with a fresh state that we know is valid.
 
 ## Supervision
 
-In Akka this "somewhere else" is called supervision. Supervision allows you to declaratively describe what should happen when a certain type of exceptions are thrown inside an actor. 
+In Akka this "somewhere else" is called supervision. Supervision allows you to declaratively describe what should happen when certain types of exceptions are thrown inside an actor. 
 
-To use supervision the actual Actor behavior is wrapped using `Behaviors.supervise`. Typically you would wrap the actor
- with supervision in the parent when spawning it as a child.
+While Akka has @ref:[supervision](../general/supervision.md) and fault tolerance built into every Actor, in many cases
```

Comment:
In Typed the default supervision is to stop the actor. In Classic it's to restart. Therefore I'm not sure the "supervision and fault tolerance built into every Actor" makes much sense here.

Maybe something like:
The default supervision strategy is to stop the actor if an exception is thrown. In many cases you will want to further customize this behavior....

Reply:
Thanks, updated.